static/css/main.css

```diff
@@ -31,3 +31,21 @@ ul {
   margin: 0;
   padding-left: 1em;
 }
+
+@media only screen and (max-device-width : 450px) {
```

Comment:
Use `max-width` instead.
`max-device-width` does not respond to browser resize.